loops-and-arrays.js

```diff
@@ -12,17 +12,30 @@ const names = ["Henk", "Piet", "Fred", "Joop"];
 // Na jouw script zie je de aangepaste waardes:
 // console.log(names) geeft: ["Henkje", "Pietje", "Fredje", "Joopje"]
 
-
-
-
+console.log(names);
+for (i = 0 ; i <names.length; i++) {
+    names[i] = names[i] + "je";
+}
+console.log(names);
+console.log();
 // ==========================================
 // Opdracht 2
 // Schrijf een script dat ieder EVEN getal in onderstaande array met 2 vermenigvuldigd, en ieder ONEVEN getal met 3
 // Zorg ervoor dat dit ook zou werken als de array wel 100 of 200 getallen zou bevatten!
 // ==========================================
 
 const numbers = [2, 4, 5, 29, 38];
+console.log(numbers);
+for (j = 0 ; j < numbers.length ; j++){
+    if (numbers[j] % 2 == 0){
+       numbers[j] = numbers[j] * 2;
+    } else {
+        numbers[j] = numbers[j] * 3;
+    }
 
+}
+console.log(numbers);
+console.log();
```

Comment:
de lege console log kan weer weg. Er mist een spatie bij for en if tussen ) en {